mongodbatlas/resource_mongodbatlas_backup_compliance_policy_test.go

```diff
@@ -0,0 +1,292 @@
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckMongoDBAtlasBackupCompliancePolicyExists(resourceName),
+					testAccCheckMongoDBAtlasBackupCompliancePolicyExists("data.mongodbatlas_backup_compliance_policy.backup_policy"),
+					resource.TestCheckResourceAttr("mongodbatlas_backup_compliance_policy.backup_policy_res", "copy_protection_enabled", "false"),
+					resource.TestCheckResourceAttr("mongodbatlas_backup_compliance_policy.backup_policy_res", "encryption_at_rest_enabled", "false"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccBackupRSBackupCompliancePolicy_importBasic(t *testing.T) {
+	var (
+		resourceName   = "mongodbatlas_backup_compliance_policy.backup_policy_res"
+		projectName    = fmt.Sprintf("testacc-project-%s", acctest.RandString(10))
+		orgID          = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectOwnerID = os.Getenv("MONGODB_ATLAS_PROJECT_OWNER_ID")
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckMongoDBAtlasBackupCompliancePolicyDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMongoDBAtlasBackupCompliancePolicyConfig(projectName, orgID, projectOwnerID),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportStateIdFunc:       testAccCheckMongoDBAtlasBackupCompliancePolicyImportStateIDFunc(resourceName),
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{""},
+			},
+		},
+	})
+}
+
+func testAccCheckMongoDBAtlasBackupCompliancePolicyExists(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		conn := testAccProvider.Meta().(*MongoDBClient).Atlas
+
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("not found: %s", resourceName)
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("no ID is set")
+		}
+
+		ids := decodeStateID(rs.Primary.ID)
+		projectID := ids["project_id"]
+
+		schedule, _, err := conn.BackupCompliancePolicy.Get(context.Background(), projectID)
+		if err != nil || schedule == nil {
+			return fmt.Errorf("Backup Compliance Policy (%s) does not exist: %s", rs.Primary.ID, err)
+		}
+
+		return nil
+	}
+}
+
+func testAccCheckMongoDBAtlasBackupCompliancePolicyDestroy(s *terraform.State) error {
+	conn := testAccProvider.Meta().(*MongoDBClient).Atlas
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "mongodbatlas_backup_compliance_policy" {
+			continue
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("no ID is set")
+		}
+
+		ids := decodeStateID(rs.Primary.ID)
+		projectID := ids["project_id"]
+
+		compliancePolicy, _, err := conn.BackupCompliancePolicy.Get(context.Background(), projectID)
+		if compliancePolicy != nil || err == nil {
+			return fmt.Errorf("backup compliance policy (%s) still exists", rs.Primary.ID)
```

Comment:
Backup Compliance Policy is capitalized like that in other instances, so for consistency: 
```suggestion
			return fmt.Errorf("Backup Compliance Policy (%s) still exists", rs.Primary.ID)
```

Reply:
Fixed